### PR TITLE
Update info.json

### PR DESCRIPTION
**Pull request by RoneyVM**
Update info.json

### CODE REVIEW

**Review by RoneyVM — COMMENTED** [review]
Update info.json

**Comment by RoneyVM** [comment]
Hello, Follow the questionnaire answered


What is the purpose of this token?
The purpose of the ShieldTokenCoin (0STC) token is to serve as the native cryptocurrency within the ShieldTokenCoin ecosystem. It has multiple use cases, including being a medium of exchange on the decentralized exchange platform, facilitating seamless and secure transactions. Additionally, the token can be used to pay for transaction fees within the blockchain network, benefiting from its low fee structure. The integration of a credit card feature enables users to directly spend their ShieldTokenCoin (0STC) holdings at establishments worldwide, making it a versatile and practical digital asset.

How will it benefit Trust Wallet users?
Trust Wallet users will benefit from the inclusion of ShieldTokenCoin (0STC) within the wallet's supported tokens. By adding support for ShieldTokenCoin (0STC), Trust Wallet enables users to store, manage, and transact with this cryptocurrency directly within the wallet's interface. This provides a seamless and secure user experience, empowering Trust Wallet users to access and utilize the features and benefits offered by ShieldTokenCoin (0STC). The integration of ShieldTokenCoin (0STC) expands the range of options and opportunities available to Trust Wallet users, promoting broader financial inclusion and access to innovative blockchain technologies.


How is the token being distributed?
Are you conducting an airdrop?
Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
Is the token listed on a centralized/decentralized exchange?
Is a Smart Contract audit completed?

How is the token being distributed?
It is being distributed through our website, where we have created a small Airdrop Event. And it can be purchased from PancakeSwap and UniSwap. It can be traded on AzBIT and VinDAX exchanges

Are you conducting an airdrop?
We are also negotiating shipping via Airdrop with:
https://airdropalert.com/
https://airdrops.io/

Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
We didn't do ICO

Is the token listed on a centralized/decentralized exchange?
Token is listed on exchanges
https://pancakeswap.finance/
https://uniswap.org/
AzBIT : https://azbit.com/exchange/0STC_USDT/
VinDAX:https://vindax.com/m-exchange-advanced.html?symbol=0STC_USDT

Is a Smart Contract audit completed?
We are in contact with CERTIK, but they have not yet started the audit

Where can we find this information?
https://shieldtokencoin.org/


RONEY VELOSO MOREIRA
Founder of ShieldTokenCoin
+5511991193256
***@***.******@***.***>
***@***.******@***.***>
________________________________
De: CryptoA ***@***.***>
Enviado: quarta-feira, 12 de julho de 2023 13:16
Para: trustwallet/assets ***@***.***>
Cc: Roney Veloso Moreira ***@***.***>; Author ***@***.***>
Assunto: Re: [trustwallet/assets] Update info.json (PR #25771)


• What is the purpose of this token? How will it benefit Trust Wallet users?
• How is the token being distributed? Are you conducting an airdrop?
• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
• Is the token listed on a centralized/decentralized exchange?
• Is a Smart Contract audit completed? Where can we find this information?

**Comment by Cryptocool1** [comment]
We require smart contract audit, please see our documentation . On hold for now

**Comment by RoneyVM** [comment]
> We require smart contract audit, please see our documentation . On hold for now

We have the Cyberscope.io audit

**Comment by RoneyVM** [comment]
We have the Cyberscope.io audit
________________________________
De: CryptoA ***@***.***>
Enviado: quarta-feira, 12 de julho de 2023 15:55
Para: trustwallet/assets ***@***.***>
Cc: Roney Veloso Moreira ***@***.***>; Author ***@***.***>
Assunto: Re: [trustwallet/assets] Update info.json (PR #25771)


We require smart contract audit, please see our documentation . On hold for now

**Review by mosesojero — COMMENTED** [review]
Moses Okeoghene Ojero

**Comment by RoneyVM** [comment]
Holders:
12,482 addresses

**Comment by RoneyVM** [comment]
Holders:
12,535 addresses
________________________________
Transfers:
38,626

________________________________
De: merge-fee-bot[bot] ***@***.***>
Enviado: quinta-feira, 13 de julho de 2023 20:31
Para: trustwallet/assets ***@***.***>
Cc: Roney Veloso Moreira ***@***.***>; Author ***@***.***>
Assunto: Re: [trustwallet/assets] Update info.json (PR #25771)


PR Summary

Files OK: 2
Token in PR: BEP20 0x3fa52142b7836468aed78753e6325b2aee7ddafe
Symbol: 0STC decimals: 18
Links: Logo<https://raw.githubusercontent.com/RoneyVM/assets/master/blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/logo.png> Info<https://raw.githubusercontent.com/RoneyVM/assets/master/blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/info.json> Website<https://shieldtokencoin.org/> Explorer<https://bscscan.com/token/0x3fa52142b7836468aed78753e6325b2aee7ddafe>
Tags: deflationary
Token check error:
❌ number of holders not checked: please, check it manually

[https://raw.githubusercontent.com/RoneyVM/assets/master/blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/logo.png]<https://raw.githubusercontent.com/RoneyVM/assets/master/blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/logo.png>

**Comment by pvcmetaofficial** [comment]
Hello hello @Cryptocool1 please reply

pease check this pool https://github.com/trustwallet/assets/pull/25799

We already payment 500 TWT as Fee and provide all details

**Comment by Cryptocool1** [comment]
Hello, is Certik audit still ongoing?

**Comment by RoneyVM** [comment]
> Hello, is Certik audit still ongoing?

We have the Cyberscope.io audit

**Comment by RoneyVM** [comment]
> We have Holders: 22,170 addresses
> 
> you can check here https://bscscan.com/token/0x3fa52142b7836468aed78753e6325b2aee7ddafe

We have Holders: 22,170 addresses

you can check here
https://bscscan.com/token/0x3fa52142b7836468aed78753e6325b2aee7ddafe

**Comment by Cryptocool1** [comment]
Hello, are you listed on CoinMarketCap?

**Comment by RoneyVM** [comment]
> Hello, are you listed on CoinMarketCap?

We haven't been officially listed yet

But here is the automatically generated page link

https://coinmarketcap.com/dexscan/pt-br/bsc/0x0b59862055c43810f4e5e2acc1109ae3f2d7be3c/

**Comment by Cryptocool1** [comment]
We require CoinMarketCap listing. Feel free to create new Pull Request once it is done. Make sure you mention this Pull Request so the fee can be reused.

**Comment by RoneyVM** [comment]
https://www.coingecko.com/en/coins/shieldtokencoin

File: blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "ShieldTokenCoin",
+    "type": "BEP20",
+    "symbol": "0STC",
+    "decimals": 18,
+    "website": "https://shieldtokencoin.org/",
```
Comment (mosesojero) [inline blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/info.json:6]: blockchains/smartchain/assets/0x3fa52142b7836468aed78753e6325b2aee7ddafe/info.json

**Pull request merged**